ci-operator/step-registry/baremetalds/assisted/test/baremetalds-assisted-test-commands.sh

```diff
@@ -29,6 +29,11 @@ scp "${SSHOPTS[@]}" "${SHARED_DIR}/test-list" "root@${IP}:/tmp/test-list"
 
 echo "### Running tests"
 timeout --kill-after 10m 120m ssh "${SSHOPTS[@]}" "root@${IP}" bash - << EOF
+    if [ -z "\$(ls -A \${KUBECONFIG})" ]; then
+        echo "Directory \${KUBECONFIG} is empty, indicating no installation was successful!"
```

Comment:
Do we have  only good flow tests?
Want to make sure that we will not fail if test should check failure case.

Reply:
e2e is always around good flows 
at least should be